src/main/webapp/dashboard/controller/Master.controller.js

```diff
@@ -66,10 +68,65 @@ sap.ui.define(
           oBinding.sort(oSorter);
         },
 
+        onChange: function(){
+           const oView = this.getView();
+           const aSelectedItems = oView.byId("runtimeConfigTable").getSelectedItems();
+           if(aSelectedItems.length == 1){
```

Comment:
Please replace loose equality ("==") with strict equality ("===") for improved type checking.